FWCore/Utilities/interface/IndexSet.h

```diff
@@ -0,0 +1,42 @@
+#ifndef FWCore_Utilities_IndexSet_h
+#define FWCore_Utilities_IndexSet_h
```

Comment:
@Dr15Jones Thanks. I don't have any plan to use this class as in a persistent data product, so maybe it can be moved to DataFormats/Common if such a need arises? (but I don't have strong opinion)

I added some documentation.